workflows/mag.nf

```diff
@@ -676,8 +723,18 @@ workflow MAG {
             /*
             * CheckM subworkflow: Quantitative measures for the assessment of genome assembly
             */
+
+            if (params.bin_domain_classification){
+                ch_input_bins_for_checkm = ch_input_bins_for_qc
+                    .filter { meta, bins ->
+                        meta.domain in ["bacteria", "archaea", "prokarya", "organelle", "unknown"]
```

Comment:
Coudl simplify/be more permissive to exclude eukarya instead?

Reply:
`!(meta.domain in ["eukarya"])`

or 

`meta.domain !in ["eukarya"]`?

Not sure which is syntactically/stylistically correct!

Reply:
Both fine, I prefer the former.
But `meta.domain != "eukarya"` also does the trick and seems simpler

Reply:
of course... 🤦‍♂️